example-stable.sql

```diff
@@ -0,0 +1,7 @@
+reate Table: CREATE TABLE `configdata` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `var` varchar(255) NOT NULL,
+  `val` varchar(255) NOT NULL,
+  `env` varchar(255) NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8 
```

Comment:
Hi @crayfishx I meant to add this one as example for the database, but I forgot to update the read me. Would you prefer to drop this example or have the readme updated? I can update the pull request.